Request/AbstractRequestBodyParamConverter.php

```diff
@@ -151,21 +155,23 @@ protected function execute(Request $request, ParamConverter $configuration)
     }
 
     /**
-     * @return DeserializationContext
+     * @return DeserializationContext|Context
      */
     protected function getDeserializationContext()
```

Comment:
Not sure if we need them at all, when we introduce the serializer API with #1171. What do you think?